projects/plugins/jetpack/modules/videopress/class.videopress-cli.php

```diff
@@ -16,11 +21,14 @@ class VideoPress_CLI extends WP_CLI_Command {
 		 * ## EXAMPLES
 		 *
 		 * wp videopress import kUJmAcSf
+		 *
+		 * @param array $args CLI arguments.
 		 */
 		public function import( $args ) {
 			$guid          = $args[0];
 			$attachment_id = create_local_media_library_for_videopress_guid( $guid );
 			if ( $attachment_id && ! is_wp_error( $attachment_id ) ) {
+				/* translators: %s: attachment id */
```

Comment:
```suggestion
				/* translators: %d: attachment id */
```
I'm not sure how important this actually is, but I notice here and on line 70 that the '%s' in the comment doesn't match the '%d' in the code itself.